firebase/remoteconfig/v1/remote_config_event_data.go

```diff
@@ -33,9 +33,32 @@ type UpdateUser struct {
 	Name     *string `json:"name,omitempty"`    // Display name.
 }
 
-type Update struct {
+type UpdateOriginEnum string
+const (
+	AdminSDKNode UpdateOriginEnum = "ADMIN_SDK_NODE"
+	Console UpdateOriginEnum = "CONSOLE"
+	RESTAPI UpdateOriginEnum = "REST_API"
+	RemoteConfigUpdateOriginUnspecified UpdateOriginEnum = "REMOTE_CONFIG_UPDATE_ORIGIN_UNSPECIFIED"
+)
+
+type UpdateTypeEnum string
+const (
+	ForcedUpdate UpdateTypeEnum = "FORCED_UPDATE"
+	IncrementalUpdate UpdateTypeEnum = "INCREMENTAL_UPDATE"
+	RemoteConfigUpdateTypeUnspecified UpdateTypeEnum = "REMOTE_CONFIG_UPDATE_TYPE_UNSPECIFIED"
+	Rollback UpdateTypeEnum = "ROLLBACK"
+)
+
+// Where the update action originated.
+type UpdateOrigin struct {
+	Enum    *UpdateOriginEnum
+	Integer *int64
```

Comment:
What does the integer represent?

Reply:
I believe it's the numeric representation (rather than string representation) of the proto enum:

https://github.com/googleapis/google-cloudevents/blob/master/proto/google/events/firebase/remoteconfig/v1/data.proto#L62